metricshub-agent/src/main/java/org/metricshub/web/service/ConfigurationFilesService.java

```diff
@@ -450,11 +445,137 @@ public Validation validate(final String content, final String fileName) {
 		}
 	}
 
+	// === BACKUP FILE OPERATIONS ===
+	private static final String BACKUP_DIR_NAME = "backup";

```

Comment:
[Move this static variable to comply with Java Code Conventions ](https://next.sonarqube.com/sonarqube/coding_rules?open=java%3AS1213&rule_key=java%3AS1213)